packages/ai/streams/google-generative-ai-stream.ts

```diff
@@ -30,6 +30,12 @@ interface TextPart {
   inlineData?: never;
 }
 
+interface FileDataPart {
+  text?: never;
+  mimeType: string;
+  fileUri: string;
+}
+
```

Comment:
please revert - this is a legacy provider that we are not maintaining and that will be removed soon